quark/subproject.py

```diff
@@ -198,7 +198,7 @@ def add_module(parent, name, uri, options, conf, **kwargs):
         while len(stack):
             current_module = stack.pop()
             if current_module.external_project:
-                generate_cmake_script(current_module.directory, update = update)
+                generate_cmake_script(current_module.directory, update = update, clean = clean, clobber = clobber, fix_remotes=fix_remotes)
```

Comment:
I see you added the previously missing `clean` and `clobber` parameters. Logically it seems correct to me, but is this a fix for a previous bug? If so, it should go in a separate commit.

Reply:
Yeah, it's essentially a bugfix; I thought about this, but I was lazy. I'll split it out now.